scenes/ink_combat/ink_combat.gd

```diff
@@ -19,7 +19,16 @@ func _ready() -> void:
 		node.completed.connect(_on_inkwell_completed)
 
 
+func _remove_enemies():
+	for node: Node in on_the_ground.get_children():
+		if node is not InkDrinker:
```

Comment:
Maybe we can use groups here? Groups allow for querying independently of the hierarchy

Reply:
Oh yes, in fact we have one already!